specs/models/ContentObject.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://marklogic.com/data-hub/ContentObject.schema.json",
+  "title": "ContentObject",
+  "description": "Defines the object that is passed, either by itself or within an array, to a step module for processing and is then used for writing documents",
+  "type": "object",
+  "properties": {
+    "uri": {
```

Comment:
There is also an optional `previousUri` property that can be a string or array of strings. It is used for the derivedFrom coarse-grained document-level provenance. The mastering/merging steps are the only ones that use it at the moment.

Reply:
I noticed that one, but I'm thinking that isn't worth including yet until we can explain to a user what the result of populating that would be - i.e. why would a user populate this? 

I also noticed "$delete", I need to add that, as that's an easy one to explain.

Reply:
It may be worth changing previousUri to derivedFrom at some point. That would be easier to explain the provenance connection.